pynsee/utils/save_df.py

```diff
@@ -44,15 +44,9 @@ def wrapper(*args, **kwargs):
             else:
                 file_name += ".pkl"            
                                     
-            if any([(a == 'update') & (kwargs[a] == True) for a in kwargs.keys()]):
-                update = True
-            else:
-                update = False  
+            update = any((a == 'update') and (kwargs[a] is True) for a in kwargs)
```

Comment:
After suggesting the change to a single liner, I think I know realize that this should be even simpler, shouldn't it?
```suggestion
            update = kwargs.get("update", False)
```